### ISSUE TITLE
YoloMouse disables GSync

### ISSUE THREAD

**Issue by Aussiemon** [PandaTeemo/YoloMouse#19]
Version 0.11.3 x64, Windows 10 Enterprise v2004, Nvidia 2080TI

Having YoloMouse enabled disables GSync. The GSync Indicator overlay does not appear, and screen tearing indicates it's fully disabled. Reproduced in a random assortment of games (e.g. Garry's Mod, Guild Wars 2, Dragon's Dogma). Closing YoloMouse brings the indicator back, and opening YoloMouse removes it. Running YoloMouse as Administrator does not seem to make a difference.

With YoloMouse in Guild Wars 2:
![Capture](https://user-images.githubusercontent.com/13663574/89114963-53ab8800-d43f-11ea-9326-54f0bd5301ae.PNG)

Without YoloMouse:
![Capture2](https://user-images.githubusercontent.com/13663574/89114965-5b6b2c80-d43f-11ea-8e15-548dc2e5f74b.PNG)

**Comment by PandaTeemo** [comment]
Thanks for the report. This is most likely fixed with the latest version
since a major rework has been done in that area including elimination of
refresh syncing. However I cannot confirm as I still have a crappy 60hz
monitor :)

**Comment by Aussiemon** [comment]
I've just purchased 1.0.3 (non-Steam edition) and I can verify that the GSync Indicator overlay is still missing.

Let me know if there's any way I can provide assistance through additional information or testing!

**Comment by PandaTeemo** [comment]
This may require too much back/forth to implement without having a GSync
monitor of my own. However next update the overlay part of YM that causes
will become optional. I assume this issue is only happening when you're
going fullscreen or is this also problem in windowed or borderless?

And thanks for the support <3

**Comment by Aussiemon** [comment]
Yes, only fullscreen it seems. In Dragon's Dogma, with the game set to windowed and GSync set to "Windowed + Fullscreen" mode, the indicator overlay still appears with YM enabled; and disappears when the game is set back to fullscreen.

**Comment by nakami** [comment]
Just purchased the program. Sad to find out about this incompatibility afterwards...

**Comment by PandaTeemo** [comment]
this is on priority TODO list. need to get a gsync monitor to fully solve
it ;)

for now you can avoid using overlay cursors (ctrl alt 2) which trigger the
overlay that causes the issue.
